game.py

```diff
@@ -20,8 +20,34 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses  = build_letter_status_dict(snowman_word)
```

Comment:
```suggestion
    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
```

Remove extra whitespace. Small inconsistencies in a codebase add up and they make the code more difficult to read/understand and harder to maintain (update/refactor). 

Code that is ready for review, like what is here in this PR, should comply with style conventions as described by the official Python style guide. 

You can read more about whitespaces [here](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements).